src/common/screens/reducer.js

```diff
@@ -4,7 +4,7 @@ import {Record} from 'immutable';
 // We can use simple initialState if no data from server need to be revived.
 const InitialState = Record({
   currentScreen: 0,
-  lastScreen: 5
+  lastScreen: 8
```

Comment:
I'm writing a comment to this and your previous comments here because the problem is the same - many parts of this code are much more complicated than they should be, some problems are solved in a really weird way - it's hard to change it sometimes. I was doing refactoring of some parts but often I must just keep this architecture to be sure I'm not breaking other things. Now we are going to make a new design of this app so I stopped changing it until I get a project of this new version.